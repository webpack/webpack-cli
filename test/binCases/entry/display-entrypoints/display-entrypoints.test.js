"use strict";

const { run, extractSummary } = require("../../../testUtils");

test("display-entrypoints", () => {
	const { stdout, stderr } = run(__dirname, ["--display-entrypoints", "false"]);

	const summary = extractSummary(stdout);

	expect(summary).not.toContain("Entrypoint");
	expect(stderr).toHaveLength(0);
});
