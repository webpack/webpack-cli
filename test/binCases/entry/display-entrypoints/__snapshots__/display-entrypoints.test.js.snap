// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display-entrypoints 1`] = `
"  Asset       Size  Chunks             Chunk Names
main.js  947 bytes       0  [emitted]  main
[0] ./index.js 26 bytes {0} [built]

WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/concepts/mode/"
`;
